open_diloco/train_fsdp.py

```diff
@@ -228,11 +214,11 @@ def train(config: Config):
         log_visible_maddrs(dht.get_visible_maddrs(), only_p2p=False)
 
     if local_rank == 0:
-        check_checkpoint_path_access(config.checkpoint_path, rank, config.hv.world_rank if config.hv else None)
+        check_checkpoint_path_access(config.ckpt.path, rank, config.hv.world_rank if config.hv else None)
 
     # DataLoader preparation
     tokenizer = AutoTokenizer.from_pretrained("mistralai/Mistral-7B-v0.1", use_fast=True)
-    tokenizer.pad_token = "</s>"  # Ensure pad token is set for models that need it
+    tokenizer.pad_token = " { "  # Ensure pad token is set for models that need it
```

Comment:
is there a reason for changing the pad token? this seems like a weird pad token to me

Reply:
wtf good catch. I think it is coming from cursor refactoring stuff haha